muffnn/mlp/base.py

```diff
@@ -286,31 +314,6 @@ def _set_up_graph(self):
             **self.solver_kwargs if self.solver_kwargs else {}).minimize(
             self._obj_func)
 
-    def _make_feed_dict(self, X, y=None):
-        # Make the dictionary mapping tensor placeholders to input data.
-
-        if self.is_sparse_:
-            indices, values = _sparse_matrix_data(X)
-
-            feed_dict = {
-                self._input_indices: indices,
-                self._input_values: values,
-                self._input_shape: X.shape
-            }
-        else:
-            feed_dict = {
-                self._input_values: X
-            }
-
-        if y is None:
-            # If y is None, then we are doing prediction and should fix
-            # dropout.
-            feed_dict[self._keep_prob] = 1.0
-        else:
-            feed_dict[self.input_targets_] = y
-            feed_dict[self._keep_prob] = self.keep_prob
-
-        return feed_dict
```

Comment:
We need to keep this function in order to control the behavior for dropout and predictions. We can adopt it to build the proper feed dictionaries for various cases.